extensions/eo/README.md

```diff
@@ -22,20 +27,22 @@ For defining view geometry of data, it is strongly recommended to use the [`view
   - [Landsat 8 with bands in Item Asset Definition and Collection Summaries](../item-assets/examples/example-landsat8.json)
 - [JSON Schema](json-schema/schema.json)
 
-## Item fields
+## Item Properties or Item Asset fields
```

Comment:
I would like to go through all of the extensions and make this consistent.  I found it confusing when this was "Item" because that could be interpreted to mean that it's a new attribute on Item, rather than it can be added to Item Properties or Item Assets. I looked a a few of the other extensions and some have "Item" and some "Item Properties" (and maybe those also apply to Assets?). Other extensions have this heading as the Object the properties go in, e.g., Collection or Catalog.

Reply:
+1 I'll put up a ticket to track. We only recently added the ability to flexibly put things on assets. I think technically people can put any property field on an asset, but we should call out in each extension where we think that's appropriate.